pkg/cfn/manager/iam.go

```diff
@@ -18,7 +21,7 @@ func (c *StackCollection) makeIAMServiceAccountStackName(namespace, name string)
 }
 
 // createIAMServiceAccountTask creates the iamserviceaccount in CloudFormation
-func (c *StackCollection) createIAMServiceAccountTask(errs chan error, spec *api.ClusterIAMServiceAccount, oidc *iamoidc.OpenIDConnectManager) error {
+func (c *StackCollection) createIAMServiceAccountTask(errs chan error, spec *api.ClusterIAMServiceAccount, oidc *iamoidc.OpenIDConnectManager, replaceIfExists bool) error {
```

Comment:
nitpicky but can this be called `replaceExistingRole`? Took me slightly more than Ctrl-F ing to find where that variable from `NewTasksToCreateIAMServiceAccounts` was actually used

Reply:
Well spotted.